lib/signs_ui/application.ex

```diff
@@ -16,7 +16,13 @@ defmodule SignsUi.Application do
       {SignsUi.Signs.State, [name: SignsUi.Signs.State]},
       SignsUi.Config.Expiration,
       SignsUi.RefreshTokenStore,
-      SignsUi.Alerts.State
+      {Application.get_env(:signs_ui, :alert_producer),
+       name: AlertProducer,
+       url: "#{System.get_env("API_V3_HOST")}/alerts?filter[datetime]=NOW&filter[route_type]=0,1",
```

Comment:
Gotta update here, too. ;) And the README, looks like.

Reply:
Wish we could have caught that with tests. Maybe this needs integration tests?

Reply:
Hm, testing ENV vars is hard. Even an integration test would probably run with `MIX_ENV=test`, I'd think. 🤔 

Our dev environment would have caught it, and we check the logs and click around there before deploying... in some sense that's like an integration test! I'd be open to a better way, but I can't think of how we could have caught this specific failure automatically.